lib/systemd.c

```diff
@@ -86,6 +86,7 @@ static int systemd_get_watchdog_time(int the_process)
 
 void systemd_send_stopping(void)
 {
+	systemd_send_information("STATUS=");
```

Comment:
maybe `"stopping %s"` here as well?

Reply:
nah.  status stays there *after* you've stopped. so a `systemct status frr` would have a status of `stopping` after it was stopped, which I found confusing.

Reply:
ah 🙂